pilot/pkg/model/push_context_test.go

```diff
@@ -649,6 +651,168 @@ func TestSidecarScope(t *testing.T) {
 	}
 }
 
+func TestBestEffortInferServiceMTLSMode(t *testing.T) {
+	const testNamespace string = "test-namespace"
```

Comment:
make `beta-namespace` and `new-namespace` as constant var as well to be consistent

Reply:
clearer they're different ns.

Reply:
done